federation/registry/cluster/strategy_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/kubernetes/federation/apis/federation"
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/testapi"
```

Comment:
See an example [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/node/strategy_test.go#L50), the call to `api.Registry.GroupOrDie(api.GroupName)` would die if no group registered(it won't without importing `pkg/api/testapi`).